account_export_csv/wizard/account_export_csv.py

```diff
@@ -65,12 +59,17 @@ class AccountCSVExport(models.TransientModel):
     date_range_id = fields.Many2one(
         comodel_name='date.range', string='Date range')
     journal_ids = fields.Many2many(
-        comodel_name='account.journal', relation='rel_wizard_journal',
-        column1='wizard_id', column2='journal_id', string='Journals',
+        comodel_name='account.journal', string='Journals',
+        default=lambda s: s._get_journal_default(),
         help='If empty, use all journals, only used for journal entries')
     export_filename = fields.Char(
         string='Export CSV Filename', size=128, default='account_export.csv')
 
+    @api.model
+    def _get_journal_default(self):
+        """ Implements your own default """
+        return False
```

Comment:
Nit: Is it really useful to have this method, compared to letting people override with Odoo standard mechanisms?

Reply:
Nit: Also for company, is it not better to use `_company_default_get`?